### PR TITLE
Minor improvements to network code

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: MediaBrowser.Api/EnvironmentService.cs [hunk 1]
```diff
@@ -52,6 +52,7 @@ public class ValidatePath
         public bool? IsFile { get; set; }
     }
 
+    [Obsolete]
```
Comment (JustAMan) [inline MediaBrowser.Api/EnvironmentService.cs]: why? there is a feature where we expose alternative path for clients who can handle that (like Kodi), is it another feature?

File: Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs [hunk 2]
```diff
@@ -123,24 +122,29 @@ public static string GetResponseContentType(HttpRequest httpReq)
                 return specifiedContentType;
             }
 
-            const string serverDefaultContentType = "application/json";
+            const string ServerDefaultContentType = "application/json";
```
Comment (cvium) [inline Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs]:
```suggestion
            const string ServerDefaultContentType = MediaTypeNames.Application.Json;
```

File: MediaBrowser.Api/EnvironmentService.cs (hunk 1 above)
Comment (cvium) [reply MediaBrowser.Api/EnvironmentService.cs]: I'm fairly certain it's a different feature.

**Comment by JustAMan** [comment]
@Bond-009 your latest commit isn't building :)

**Comment by cvium** [comment]
Missing `using System.Net.Mime;`

**Pull request merged**